paket.dependencies

```diff
@@ -1,10 +1,13 @@
 source http://nuget.org/api/v2
+framework: net451
 
+nuget FSharp.Core redirects: force
 nuget FAKE
 nuget SourceLink.Fake
 nuget FSharp.Formatting
 nuget NUnit
-nuget NUnit.Runners
-nuget WindowsAzure.ServiceBus
+nuget NUnit.Console
+nuget Microsoft.WindowsAzure.ConfigurationManager 2.0.3
+nuget WindowsAzure.ServiceBus 2.5.2
 nuget Newtonsoft.Json
-nuget Unquote
\ No newline at end of file
+nuget Unquote 2.2.2
```

Comment:
Any reason for the explicit version pins?

Reply:
If I do not pin this, it upgrades to Unquote 3 - which breaks stuff all over the place. TBH, I was to layz to check for the latest 2.* version and just took the one, that was in the old lock file 😦 


I just checked. 2.2.2 is the latest 2-mayor version.

Reply:
That's fine, makes perfect sense!